.github/workflows/sync.yml

```diff
@@ -0,0 +1,98 @@
+permissions:
+  contents: write
+
+name: API Docs Schema Nightly Sync
+on:
+  schedule:
+    - cron: '0 0 * * *' # every day at midnight UTC
+
+jobs:
+  sync_api_docs_with_binary_websocket_api:
+    name: Nightly Sync API Docs with Binary Websocket API
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          path: deriv-api-docs
+          ref: master
+
+      - name: Checkout binary-websocket-api Repository
+        uses: actions/checkout@v4
+        with:
+          repository: regentmarkets/binary-websocket-api
+          path: deriv-websockets-api
+          token: ${{ secrets.API_SYNC_PAT }} # TODO: Replace with correct Personal Access Token when provided
+          ref: master
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '21'
+
+      - name: Install Dependencies
+        run: |
+          npm install yaml
+          npm install fs-extra
+
+      - name: Configure Git
+        run: |
+          git config --global user.name "PUT NAME HERE WHEN PROVIDED"
+          git config --global user.email "put_email_here@when_provided.com"
+
+      - name: Copy Files From Source to Destination
+        run: node deriv-api-docs/.github/scripts/get-web-socket-api-list.js
+
+      - name: Generate Branch Name
+        id: generateBranchName
+        run: echo "BRANCH=update-schema_$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        id: createPullRequest
+        uses: peter-evans/create-pull-request@v5
+        with:
+          commit-message: JSON Schema Update
+          title: JSON Schema Update
+          body: JSON Schema Update
+          branch: ${{ steps.generateBranchName.outputs.BRANCH }}
+          base: master
+          token: ${{ secrets.TEST_API_PR_PAT }} # TODO: Replace with correct Personal Access Token when provided
+          path: deriv-api-docs
+          delete-branch: true
+
+      - name: Merge Pull Request
```

Comment:
It looks like we have checks in place. I think we've to wait for those checks to be passed before merging.
<img width="913" alt="image" src="https://github.com/binary-com/deriv-api-docs/assets/56330681/474628d9-51f2-4a4c-96db-dea7cca8747f">